[debug] [2025-06-21T14:49:45.615Z] ----------------------------------------------------------------------
[debug] [2025-06-21T14:49:45.616Z] Command:       /home/suchitra/.config/nvm/versions/node/v20.19.2/bin/node /usr/bin/firebase init
[debug] [2025-06-21T14:49:45.617Z] CLI Version:   14.7.0
[debug] [2025-06-21T14:49:45.617Z] Platform:      linux
[debug] [2025-06-21T14:49:45.617Z] Node Version:  v20.19.2
[debug] [2025-06-21T14:49:45.617Z] Time:          Sat Jun 21 2025 20:19:45 GMT+0530 (India Standard Time)
[debug] [2025-06-21T14:49:45.617Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-06-21T14:49:45.619Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2025-06-21T14:49:45.657Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-06-21T14:49:45.658Z] > authorizing via signed-in user (suchitraidumina@gmail.com)
[info] 
     ######## #### ########  ######## ########     ###     ######  ########
     ##        ##  ##     ## ##       ##     ##  ##   ##  ##       ##
     ######    ##  ########  ######   ########  #########  ######  ######
     ##        ##  ##    ##  ##       ##     ## ##     ##       ## ##
     ##       #### ##     ## ######## ########  ##     ##  ######  ########

You're about to initialize a Firebase project in this directory:

  /home/suchitra/Mystuff/Aiteera/Projects/salesGPT

[debug] [2025-06-21T14:49:46.349Z] <<< [apiv2][status] GET https://firebase-public.firebaseio.com/cli.json 200
[debug] [2025-06-21T14:49:46.349Z] <<< [apiv2][body] GET https://firebase-public.firebaseio.com/cli.json {"cloudBuildErrorAfter":1594252800000,"cloudBuildWarnAfter":1590019200000,"defaultNode10After":1594252800000,"minVersion":"3.0.5","node8DeploysDisabledAfter":1613390400000,"node8RuntimeDisabledAfter":1615809600000,"node8WarnAfter":1600128000000}
[debug] [2025-06-21T14:55:54.058Z] ExitPromptError: User force closed the prompt with SIGINT
    at Interface.sigint (/usr/lib/node_modules/firebase-tools/node_modules/@inquirer/core/dist/commonjs/lib/create-prompt.js:101:37)
    at Interface.emit (node:events:524:28)
    at Interface.emit (node:domain:489:12)
    at [_ttyWrite] [as _ttyWrite] (node:internal/readline/interface:1125:18)
    at ReadStream.onkeypress (node:internal/readline/interface:264:20)
    at ReadStream.emit (node:events:536:35)
    at ReadStream.emit (node:domain:489:12)
    at emitKeys (node:internal/readline/utils:371:14)
    at emitKeys.next (<anonymous>)
    at ReadStream.onData (node:internal/readline/emitKeypressEvents:64:36)
[error] 
[error] Error: An unexpected error has occurred.
